nav2_simple_navigator/src/simple_navigator.cpp

```diff
@@ -57,14 +119,14 @@ SimpleNavigator::navigateToPose(const nav2_tasks::NavigateToPoseCommand::SharedP
   // Create the path to be returned from ComputePath and sent to the FollowPath task
   auto path = std::make_shared<nav2_tasks::ComputePathToPoseResult>();
 
-  RCLCPP_DEBUG(get_logger(), "Getting the path from the planner");
-  RCLCPP_DEBUG(get_logger(), "goal->pose.position.x: %f", command->pose.position.x);
-  RCLCPP_DEBUG(get_logger(), "goal->pose.position.y: %f", command->pose.position.y);
-  RCLCPP_DEBUG(get_logger(), "goal->pose.position.z: %f", command->pose.position.z);
-  RCLCPP_DEBUG(get_logger(), "goal->pose.orientation.x: %f", command->pose.orientation.x);
-  RCLCPP_DEBUG(get_logger(), "goal->pose.orientation.y: %f", command->pose.orientation.y);
-  RCLCPP_DEBUG(get_logger(), "goal->pose.orientation.z: %f", command->pose.orientation.z);
-  RCLCPP_DEBUG(get_logger(), "goal->pose.orientation.w: %f", command->pose.orientation.w);
+  RCLCPP_DEBUG(get_logger(), "Getting the path from the planner for goal pose:");
+  RCLCPP_DEBUG(get_logger(), "  position.x: %f", command->pose.position.x);
```

Comment:
Are the leading spaces necessary here in these strings? Are you trying to show indentation? Maybe "--" would work better?

Reply:
Sure. I'll update.